.github/workflows/trivy.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Run Trivy vulnerability scanner in repo mode
-        uses: aquasecurity/trivy-action@0.24.0
+        uses: aquasecurity/trivy-action@0.25.0
```

Comment:
`v0.26.0` was released ~40 minutes ago: https://github.com/aquasecurity/trivy-action/releases/tag/0.26.0, lets update to latest as it may help with the failures we have been observing recently: https://github.com/aquasecurity/trivy-action/pull/399

Reply:
Great timing.